enhancements/ingress/optional-ingress-hypershift.md

```diff
@@ -0,0 +1,323 @@
+
+## Motivation
+
+Users of the ROSA managed OpenShift service would like to use [AWS load balancers](/enhancements/ingress/aws-load-balancer-operator.md) as a way to accept the ingress traffic into the cluster.
+The cluster ingress operator becomes unnecessary and can be disabled to:
+- reduce the resource consumption on
+  - the HyperShift user clusters (ingress controllers)
+  - the HyperShift hosted control plane (operator)
+- make the HyperShift hosted control plane lighter and simpler, therefore more manageable
+- enable the HyperShift cluster users to make their choice for the cluster ingress
+
+### Goals
+- Implement the new ingress capability.
+- Add the ingress capability to the ingress operator's payload.
+- Make the dependent cluster operators tolerate the absence of the ingress operator.
+- Do not lock the subsequent implementation for the standalone OpenShift.
+
+### Non-Goals
+- Use a different way for enabling/disabling the ingress other than via [the cluster capability](/enhancements/installer/component-selection.md).
+- Fully implement the ingress capability for the standalone OpenShift:
+  - Make the OpenShift installer tolerate the absence of the default ingress controller.
+  - Make the cluster authentication operator tolerate the absence of the default ingress controller.
+  - Make the cluster monitoring operator tolerate the absence of the default ingress controller.
+- Disable the Route API.
+- Tolerate the unadmitted routes for any workload except for the dependent cluster operators (see [Goals](#goals)).
+- Design the capability API for HyperShift.
+- Implement the ingress operator disabling in the HyperShift's control plane operator.
+
+## Proposal
+
+### User Stories
+
+- As a HyperShift engineer, I want to be able to disable the ingress operator on clusters that do not need it, so that the hosted control-plane can become lighter.
+- As a HyperShift engineer, I want cluster operators to tolerate the absence of the ingress operator and default router deployment, so that clusters do not report degraded status in the absence of these components.
+- As a HyperShift hosted cluster's user, I want the ingress capability to be reported as a known capability, so that cluster actors (admins, users, other operators) can check whether it is enabled for the cluster.
+- As a HyperShift hosted cluster's user, I want to avoid provisioning unnecessary infrastructure resources, such as an ingress ELB, when they are not needed.
+
+### Workflow Description
+
+A HyperShift Cluster User with a hosted cluster wants to use an AWS ALB to route the traffic to an application deployed on that cluster.
+
+#### Terminology
+[HyperShift concepts and personas](https://hypershift-docs.netlify.app/reference/concepts-and-personas/).
```

Comment:
This isn't a permalink.  Could you copy and paste the relevant definitions into the EP?

Reply:
Done.